frontend/javascripts/oxalis/view/action-bar/view_modes_view.tsx

```diff
@@ -51,24 +72,26 @@ class ViewModesView extends PureComponent<Props, {}> {
   }
 
   render() {
+    const handleMenuClick: MenuProps["onClick"] = (args) => {
+      if (ViewModeValues.includes(args.key as ViewMode)) {
+        this.handleChange(args.key as ViewMode);
+      }
+    };
+
+    const menuProps = {
+      items: MENU_ITEMS,
+      onClick: handleMenuClick,
+    };
+
     return (
-      <Select
-        value={this.props.viewMode}
-        style={{
-          width: 120,
-        }}
-        onChange={this.handleChange}
-      >
-        {[
-          constants.MODE_PLANE_TRACING,
-          constants.MODE_ARBITRARY,
-          constants.MODE_ARBITRARY_PLANE,
-        ].map((mode) => (
-          <Option key={mode} disabled={this.isDisabled(mode)} value={mode}>
```

Comment:
What happened to the `disabled` prop? The new dropdown will always show all the modes, right?

Reply:
Oh, very good point :+1:

Reply:
fixed now!